test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml

```diff
@@ -26,3 +27,5 @@ authenticator: 'PasswordAuthenticator'
 authenticator_user: cassandra
 authenticator_password: cassandra
 authorizer: 'CassandraAuthorizer'
+
+use_preinstalled_scylla: true
```

Comment:
@fgelcer please open a task to remove this from here, put it in the default for gce_config and add "false" to all places we need to specify it like artifacts and rolling upgrades.

Reply:
https://github.com/scylladb/qa-tasks/issues/1371